lib/skpm-build.js

```diff
@@ -103,13 +131,10 @@ function webpackConfig (file, isCommand) {
           ? path.join(output, 'Contents', 'Sketch')
           : path.join(output, 'Contents', 'Resources')
       },
-      plugins: (userDefinedWebpackConfig.plugins || []).concat(
-        isCommand ? new webpack.BannerPlugin({
-          banner: consolePolyfill,
-          raw: true,
-          entryOnly: true
-        }) : []
-      )
+      plugins: plugins,
+      resolve: {
```

Comment:
It should still be overridable - I did `+  var plugins = userDefinedWebpackConfig.plugins || []`